### ISSUE TITLE
Enhance icons

### ISSUE THREAD

**Issue by ogarcia** [ultrasonic/ultrasonic#308]
Convert all icons to vector to get two achievements:
- Make the app more slim
- Make the app more beautiful

**Comment by nitehu** [comment]
@ogarcia I know maybe I should use the other ImageLoader, but LegacyImageLoader throws an exception as soon as I start Ultrasonic after this change.

```
E/AndroidRuntime: FATAL EXCEPTION: Thread-5
    Process: org.moire.ultrasonic.debug, PID: 11159
    java.lang.ClassCastException: android.graphics.drawable.VectorDrawable cannot be cast to android.graphics.drawable.BitmapDrawable
        at org.moire.ultrasonic.util.LegacyImageLoader.createLargeUnknownImage(LegacyImageLoader.java:123)
        at org.moire.ultrasonic.util.LegacyImageLoader.<init>(LegacyImageLoader.java:83)
        at org.moire.ultrasonic.activity.SubsonicTabActivity.getImageLoader(SubsonicTabActivity.java:799)
        at org.moire.ultrasonic.util.FileUtil.getAlbumArtBitmap(FileUtil.java:220)
        at org.moire.ultrasonic.service.LocalMediaPlayer.updateRemoteControl(LocalMediaPlayer.java:492)
        at org.moire.ultrasonic.service.LocalMediaPlayer.setCurrentPlaying(LocalMediaPlayer.java:329)
        at org.moire.ultrasonic.service.MediaPlayerService.setCurrentPlaying(MediaPlayerService.java:250)
        at org.moire.ultrasonic.service.MediaPlayerService.play(MediaPlayerService.java:394)
        at org.moire.ultrasonic.service.MediaPlayerControllerImpl$2.accept(MediaPlayerControllerImpl.java:125)
        at org.moire.ultrasonic.service.MediaPlayerControllerImpl$2.accept(MediaPlayerControllerImpl.java:122)
        at org.moire.ultrasonic.service.MediaPlayerService$1.run(MediaPlayerService.java:122)
```

**Comment by ogarcia** [comment]
Yes. Fixed with #313
